block/manager.go

```diff
@@ -128,29 +128,6 @@ func NewManager(
 		return nil, fmt.Errorf("failed to get initial state: %w", err)
 	}
 
-	validators := []*tmtypes.Validator{}
-
-	if s.LastBlockHeight+1 == genesis.InitialHeight {
-		sequencersList := settlementClient.GetSequencersList()
-		for _, sequencer := range sequencersList {
-			tmPubKey, err := cryptocodec.ToTmPubKeyInterface(sequencer.PublicKey)
-			if err != nil {
-				return nil, err
-			}
-			validators = append(validators, tmtypes.NewValidator(tmPubKey, 1))
-		}
-
-		res, err := exec.InitChain(genesis, validators)
-		if err != nil {
-			return nil, err
-		}
-
-		updateInitChainState(&s, res, validators)
-		if _, err := store.UpdateState(s, nil); err != nil {
-			return nil, err
-		}
-	}
-
```

Comment:
I guess moving this to Start is just a refactor?

Reply:
also called now only by the aggregator (was in common flow before)

Reply:
Ah ok, in terminology, does aggregator = current active sequencer = proposer?

Reply:
terminology wise, yes.

initially aggreagator mode was only set by a flag in dymint.toml,
in this PR I've added validation to this assumption
(local key matches the proposer sequencer)